pkg/minikube/cluster/cluster.go

```diff
@@ -75,27 +76,27 @@ func StartHost(api libmachine.API, config MachineConfig) (*host.Host, error) {
 	glog.Infoln("Machine exists!")
 	h, err := api.Load(constants.MachineName)
 	if err != nil {
-		return nil, fmt.Errorf(
+		return nil, errors.Errorf(
```

Comment:
This could be errors.Wrap

(I think anytime we have an existing error object, the pattern is to call Wrap)